crates/ursa-proxy/src/core/handler.rs

```diff
@@ -83,3 +172,20 @@ pub async fn proxy_pass<C: Cache>(
     };
     StreamBody::new(ReaderStream::new(reader)).into_response()
 }
+
+async fn serve_file_handler(
```

Comment:
are we serving car files?

Reply:
No, this is for the let's encrypt challenge, `certbot` needs to be able to serve files. It can be configured like in nginx.

Reply:
nodes should only be serving content addressable

Reply:
This is the same as what NGINX is currently doing for the `certbot`. No other file is being served.

Reply:
I'll talk to Helder to see if we can improve the process of setting up the certs and if we can to avoid having nginx and the proxy serve any files. There is a thread on the networking channel from yesterday.